core.lua

```diff
@@ -89,7 +230,16 @@ function CustomTooltips_DisplayTooltip(button, macroText)
 	GameTooltip_SetDefaultAnchor(GameTooltip, button)
 	
 	GameTooltip:AddLine(heading, 1,1,1) -- Use white text, don't wrap the text
-	GameTooltip:AddLine(body, nil,nil,nil, true) -- Use the default yellow text colour, wrap the text
+	for i=1, #body do
+		-- Text is in body[i][1]
```

Comment:
You could probably use local variables here instead of comments, e.g. `local text, red, green, blue = body[i][1], ...`